modules/tinymce/src/plugins/table/main/ts/actions/ResizeHandler.ts

```diff
@@ -57,9 +57,10 @@ export const getResizeHandler = function (editor: Editor): ResizeHandler {
     const direction = TableDirection(Direction.directionAt);
     const rawWire = TableWire.get(editor);
     wire = Option.some(rawWire);
-    if (hasObjectResizing(editor) && hasTableResizeBars(editor)) {
+    if (Settings.hasObjectResizing(editor) && Settings.hasTableResizeBars(editor)) {
       const lazySizing = (table: SugarElement<HTMLTableElement>) => TableSize.get(editor, table);
-      const sz = TableResize.create(rawWire, direction, lazySizing);
+      const resizing = Settings.getColumnResizingBehaviour(editor) === 'resizetable' ? ResizeBehaviour.resizeTable() : ResizeBehaviour.preserveTable();
```

Comment:
There really should be a TinyMCE test for this part of the logic, to make sure the right mode to selected based on the setting used.